tests/e2e/test_local_runner_real_dispatch_e2e.py

```
@@ -0,0 +1,255 @@
+async def _wait_for_terminal(runner: LocalWorkflowRunner, workflow_id: str, timeout: float = 120.0):
+    """Poll get_status() until the workflow reaches a terminal state."""
+    deadline = time.monotonic() + timeout
+    while time.monotonic() < deadline:
+        status = await runner.get_status(workflow_id)
+        if status.is_terminal:
+            return status
+        await asyncio.sleep(0.5)
+    raise AssertionError(f"Workflow '{workflow_id}' did not reach a terminal state in {timeout}s")
+
+
+_SEVERITY_TOKEN = "critical-227"
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to: "_SEVERITY_TOKEN"

(S105)